bin/run_docker.py

```diff
@@ -42,6 +45,68 @@ def get_submission_image(syn: synapseclient.Synapse, submission_id: str) -> str:
     return image_id
 
 
+def make_invalid_output(file_name: str, log_file_path: str, file_content: str) -> str:
+    """
+    Creates an invalid new output file, given the original file name, log file path, and file content, and returns the path of the new file.
+    Parameters:
+        file_name (str): The name of the original file.
+        log_file_path (str): The path where the new file will be created.
+        file_content (str): The content to be written to the new file.
+    Returns:
+        invalid_file: The path of the newly created file with the invalid data.
+
+    """
+    invalid_file_name = "INVALID_" + file_name
+    invalid_file = os.path.join(log_file_path, invalid_file_name)
+    with open(invalid_file, "w") as d:
+        d.write(file_content)
+
+    return invalid_file
+
+
+def handle_outputs(output_path: str, output_file_name: str, log_text: str):
+    """
+    Handles any output files generated by the submitted Docker container:
+    Checks if any output files were generated by the container
+    run. If any were found, it will return the path to the first one found.
+    If none were found, it will return None.
+
+    Arguments:
+        output_path: The path to the output directory.
+        output_file_name: The name of the output file.
+        log_text: The log text to be updated, if an incorrect number of
+                  output files were found.
+
+    Returns:
+        output_file: The path to the output file.
+        log_text: The log text that was updated, if an incorrect number of
+                  output files were found.
+
+    """
+    # Glob any expected output files stored in the output_dir as a result of the container run
+    file_glob = glob(os.path.join(output_path, output_file_name + ".*"))
+
+    # Update the log file if there is a case where too many output files were generated, or none were generated
```

Comment:
Keep an eye out for code comments that don't add much context. In this case and the comment below it doesn't add value IMO & is additional maintenance.

The code itself should be the best indicator of what it's doing. I think the code you wrote is clear enough that these comments are not necessary.

Reply:
I don't feel like comments are a lot of maintenance now that we have AI that can auto-populate them, but I see what you mean. I've removed these in this case.